packages/passport/sdk/src/zkEvm/relayerClient.ts

```diff
@@ -56,6 +56,20 @@ type ImGetFeeOptionsResponse = JsonRpc & {
   result: FeeOption[]
 };
 
+// ImSign types
+type ImSignRequest = {
+  method: 'im_sign';
+  params: {
+    chainId: string;
+    address: string;
+    message: string;
+  }[];
+};
+
+type ImSignResponse = JsonRpc & {
+  result: string;
+};
```

Comment:
I can see that these reflect the relayer's open API spec, but why does `im_sign` accept an array of params if the response is a single string?